Capstone_EDA.ipynb

```diff
@@ -59,38 +59,38 @@
     "## 2.3 Feature Glossary\n",
```

Comment:
<p>Die Abbildung mit der Ordnerstruktur gehört aus meiner Sicht nicht in unser veröffentlichtes Notebook, liefert keinen Mehrwert.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/davidslabon/Capstone_UnlockingClimateSolutions/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Ja, das können wir natürlich rausnehmen. Überhaupt stellt das Notebook aktuell nicht den Stand dar, der am Ende da auch stehen bleiben soll, denke ich. Es ist für die Zeit der Entwicklung genau das: Ein Development-Notebook. Wenn wir fertig sind können wir das auf ein "Präsentation-Maß" reduzieren. Aber für den Moment würde ich erstmal alles drin lassen...</p>